colmi_r02_client/client.py

```diff
@@ -127,39 +132,37 @@ async def get_battery(self) -> battery.BatteryInfo:
         assert isinstance(result, battery.BatteryInfo)
         return result
 
-    async def get_realtime_heart_rate(self) -> list[int] | None:
-        return await self._poll_real_time_reading(real_time_hr.START_HEART_RATE_PACKET)
+    async def _poll_real_time_reading(
+            self, reading_type: rt_packet.RealTimeReading) -> list[int] | None:
+        start_packet = rt_packet.get_start_packet(reading_type)
+        stop_packet = rt_packet.get_stop_packet(reading_type)
 
-    async def _poll_real_time_reading(self, start_packet: bytearray) -> list[int] | None:
         await self.send_packet(start_packet)
 
         valid_readings: list[int] = []
         error = False
         tries = 0
         while len(valid_readings) < 6 and tries < 20:
             try:
-                data: real_time_hr.Reading | real_time_hr.ReadingError = await asyncio.wait_for(
-                    self.queues[real_time_hr.CMD_START_HEART_RATE].get(),
+                data: rt_packet.Reading | rt_packet.ReadingError = await asyncio.wait_for(
+                    self.queues[rt_packet.CMD_START_REAL_TIME].get(),
                     timeout=2,
                 )
-                if isinstance(data, real_time_hr.ReadingError):
+                if isinstance(data, rt_packet.ReadingError):
                     error = True
                     break
                 if data.value != 0:
                     valid_readings.append(data.value)
             except TimeoutError:
                 tries += 1
-                await self.send_packet(real_time_hr.CONTINUE_HEART_RATE_PACKET)
```

Comment:
i noticed this wasn't really doing much and everything's working as expected without it. please let me know if there was a particular reason you were sending it and i'll add it back

Reply:
It's unclear what it does, but the QRing app does send this while taking a real time measurement. Let's leave it in

Reply:
added back